### ISSUE TITLE
Unable to run py.test without having pytest_reportportal configured once installed

### ISSUE THREAD

**Issue by rplevka** [reportportal/agent-python-pytest#80]
what i'm looking for here is to make the `report_portal logger` optional - however, once the plugin gets installed, one is unable to run `py.test` without having configured `pytest.ini` (or options passed by `-o rp_*=value`).
The pytests fails with:

```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/_pytest/main.py", line 176, in wrap_session
INTERNALERROR>     config.hook.pytest_sessionstart(session=session)
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pluggy/__init__.py", line 617, in __call__
INTERNALERROR>     return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pluggy/__init__.py", line 222, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pluggy/__init__.py", line 216, in <lambda>
INTERNALERROR>     firstresult=hook.spec_opts.get('firstresult'),
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pluggy/callers.py", line 201, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pluggy/callers.py", line 76, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pluggy/callers.py", line 180, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pytest_reportportal/plugin.py", line 53, in pytest_sessionstart
INTERNALERROR>     wait_launch(session.config.py_test_service.RP.rp_client)
INTERNALERROR>   File "/home/rplevka/.virtualenvs/master_robottelo_p3/lib/python3.6/site-packages/pytest_reportportal/plugin.py", line 60, in wait_launch
INTERNALERROR>     raise Exception("Launch not found")
INTERNALERROR> Exception: Launch not found
```

- I'm not initializing the logger anywhere in my code, it's just the presence of the plugin.
- I would expect the plugin skips configuring if it can't find some vital configuration options (e.g. `endpoint`).

**Comment by rplevka** [comment]
I also tried to workaround this by implementing my own hook function in the `conftest.py`:

```python
def pytest_plugin_registered(plugin, manager):
    if 'pytest_reportportal' in str(plugin.__class__):
        manager.unregister(plugin)
```
however the `session_starts`  hook still does the setup.

**Comment by rplevka** [comment]
ok, i tihnk this got already addressed by:
https://github.com/reportportal/agent-python-pytest/pull/60

..looks like it is just not included in the `1.0.3` release
